cpp/basix/e-brezzi-douglas-marini.cpp

```diff
@@ -6,6 +6,7 @@
 #include "e-lagrange.h"
 #include "e-nedelec.h"
 #include "element-families.h"
+#include "finite-element-utils.h"
```

Comment:
Looks like `finite-element.h` is missing - the `FiniteElement` constructor is used in this file.